docs/modules/clusters/pages/partition-group-configuration.adoc

```diff
@@ -90,19 +90,21 @@ partitionGroupConfig.setEnabled( true )
 
 === ZONE_AWARE
 
-You can use ZONE_AWARE configuration with https://github.com/hazelcast/hazelcast-kubernetes[Hazelcast Kubernetes^],
-https://github.com/hazelcast/hazelcast-aws[Hazelcast AWS^],
-https://github.com/hazelcast/hazelcast-gcp[Hazelcast GCP^],
-https://github.com/hazelcast/hazelcast-jclouds[Hazelcast jclouds^] or
-https://github.com/hazelcast/hazelcast-azure[Hazelcast Azure^] Discovery Service plugins.
+You can use `ZONE_AWARE` configuration when you want to back up data on different Availability Zones (AZ) in the following environments:
 
-As discovery services, these plugins put zone information to the Hazelcast
+- xref:deploy:deploying-in-kubernetes.adoc[Hazelcast Kubernetes],
+- xref:deploy:deploying-on-aws.adoc[Hazelcast AWS],
+- xref:deploy:deploying-on-gcp.adoc[Hazelcast GCP],
+- link:https://github.com/hazelcast/hazelcast-jclouds[Hazelcast jclouds^]
+- xref:deploy:deploying-on-azure.adoc[Hazelcast Azure]
+
+These services write zone information to the Hazelcast
```

Comment:
We also need to change the link `https://github.com/hazelcast/hazelcast-kubernetes/tree/1.5.x` in the line 212